prowler/providers/gcp/services/cloudsql/cloudsql_instance_ssl_connections/cloudsql_instance_ssl_connections.py

```diff
@@ -12,12 +12,16 @@ def execute(self) -> Check_Report_GCP:
             report.resource_name = instance.name
             report.location = instance.region
             report.status = "PASS"
-            report.status_extended = (
-                f"Database Instance {instance.name} requires SSL connections."
-            )
-            if not instance.ssl:
+            report.status_extended = f"Database Instance {instance.name} requires SSL connections and allows only encrypted connections."
+            if not instance.require_ssl and instance.ssl_mode != "ENCRYPTED_ONLY":
                 report.status = "FAIL"
-                report.status_extended = f"Database Instance {instance.name} does not require SSL connections."
+                report.status_extended = f"Database Instance {instance.name} does not require SSL connections and allows unencrypted connections."
+            elif not instance.require_ssl:
+                report.status = "FAIL"
+                report.status_extended = f"Database Instance {instance.name} does not require SSL connections but allows only encrypted connections."
+            elif instance.ssl_mode != "ENCRYPTED_ONLY":
+                report.status = "FAIL"
+                report.status_extended = f"Database Instance {instance.name} requires SSL connections but allows unencrypted connections."
```

Comment:
Please leave an unique status extended when the SSL is not enabled either for one attribute or the other.